extensions/blocks/podcast-player/templates/playlist-track.php

```diff
@@ -30,16 +32,27 @@
 
 <li
 	class="<?php echo esc_attr( trim( $class ) ); ?>"
+	[class]="'<?php echo esc_attr( trim( 'jetpack-podcast-player__track ' . $secondary_colors['class'] ) ); ?> ' + (currentTrack == <?php echo esc_attr( $index ); ?> ? 'is-active' : '')"
 	style="<?php echo esc_attr( $style ); ?>"
+	option="<?php echo esc_attr( $index ); ?>"
```

Comment:
What is the `option` attribute?

Reply:
That's how you mark a selectable item in `<amp-selector>` and give it a value (here, we use the index as value)